1.9.0

```diff
@@ -771,6 +771,21 @@
           }
         }
       ]
+    },
+    {
+      "sObjectName":"Opportunity",
+      "description": "Update Stage Picklist Values",
+      "taskList":[
+        {
+          "jiraTicket":"ac-64579",
+          "actionType":"PicklistValueStandard",
+          "actionTypeName":"StageName",
```

Comment:
You're gonna love this. Apparently it is just Stage even though the api name in salesforce says StageName. I couldn't get it to run with StageName but once I changed it to just Stage it worked. What the hell.

https://developer.salesforce.com/docs/atlas.en-us.api_meta.meta/api_meta/meta_standardvalueset.htm